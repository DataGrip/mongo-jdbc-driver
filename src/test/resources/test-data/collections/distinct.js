// before
db.col.insertMany([{v: 1}, {v: 2}, {v: 3}, {v: "hello"}, {v: 1}]);
// command
db.col.distinct("v");
// clear
db.col.drop();