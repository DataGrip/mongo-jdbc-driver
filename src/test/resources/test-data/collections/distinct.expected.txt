|result|
|1|
|2|
|3|
|hello|